charts/scalardb/templates/config-map.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scalardb-config-file
+data:
+  # Create a database.properties file which is config file of Scalar DB server.
+  database.properties: |-
+    scalar.db.server.port={{.Values.scalardb.serverCfg.port}}
+    scalar.db.server.prometheus_http_endpoint_port={{.Values.scalardb.serverCfg.prometheus_http_endpoint_port}}
```

Why do you use `values.scalardb.server "Cfg".port`? In `database.properties` setting, it is `scalar.db.server.port`, so it seems to be aligned and easy to understand.

```suggestion
    scalar.db.server.port={{.Values.scalardb.server.port}}
    scalar.db.server.prometheus_http_endpoint_port={{.Values.scalardb.server.prometheus_http_endpoint_port}}
```

---

Thanks for your suggestions. I'll fix it.